standard/lexical-structure.md

```diff
@@ -703,6 +752,25 @@ If the magnitude of the specified literal is too large to be represented in the
 The value of a real literal of type `float` or `double` is determined by using the IEC 60559 “round to nearest” mode with ties broken to “even” (a value with the least-significant-bit zero), and all digits considered significant.
 
 > *Note*: In a real literal, decimal digits are always required after the decimal point. For example, `1.3F` is a real literal but `1.F` is not. *end note*
+>
+> *Example*:
+>
+> ```csharp
+> 1.234_567              // double
+> .3e5f                  // float
+> 2_345E-2_0             // double
+> 15D                    // double
+> 19.73M                 // decimal
+> 1.F                    // invalid; ill-formed (parsed as "1." and "F")
+> _1.2F                  // invalid; no leading _ allowed
+> 1_.2F                  // invalid; no trailing _ allowed in integer part
+> 1._234                 // invalid; no leading _ allowed in fraction
```

Comment:
This is a syntactically valid member selection expression.

Reply:
Change as per previous

Reply:
Done.